examples/dask/special-topics-rolling-averages.ipynb

```diff
@@ -0,0 +1,197 @@
+     "iopub.status.idle": "2022-01-25T17:33:25.666206Z",
+     "shell.execute_reply": "2022-01-25T17:33:25.665688Z",
+     "shell.execute_reply.started": "2022-01-25T17:33:22.602827Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Create Rolling Averages with Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "866d3726-027e-4c04-ae8b-717683685472",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-01-25T17:33:22.602911Z",
+     "iopub.status.busy": "2022-01-25T17:33:22.602324Z",
+     "iopub.status.idle": "2022-01-25T17:33:25.666206Z",
+     "shell.execute_reply": "2022-01-25T17:33:25.665688Z",
+     "shell.execute_reply.started": "2022-01-25T17:33:22.602827Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "\"I need to calculate a rolling average of a numerical column, in time series data. In pandas, I can do this with rolling(x).mean() with sorted values, but what do I do in Dask, with distributed data?\"\n",
+    "\n",
+    "* Sort by index within AND across partitions\n",
+    "* Know when to compute (convert to pandas DF) or persist (process computations on cluster)\n",
+    "* Run calculations, with attention to our need to cross partitions correctly.\n",
+    "\n",
+    "This example will walk you through these specific points, and demonstrate how it's done. We'll use New York City taxi trip data, and get the 30-day rolling average of base fare prices, for our example."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "622fb142-cee7-4659-a9b5-d9cbde6bf912",
+   "metadata": {},
+   "source": [
+    "## Single Node"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c831ea89-1530-404f-8acc-94d3c28a603b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "096acff8-a6a7-4df0-a971-0c18a483a783",
+   "metadata": {},
+   "source": [
+    "## Cluster"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "462c9dea-fe10-4fa4-8120-15ee89caf7ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_saturn import SaturnCluster\n",
+    "from dask.distributed import Client\n",
+    "\n",
+    "cluster = SaturnCluster(\n",
+    "    scheduler_size=\"medium\",\n",
+    "    worker_size=\"xlarge\",\n",
+    "    n_workers=3,\n",
+    "    nthreads=4,\n",
+    ")\n",
+    "client = Client(cluster)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "280f72b9-9602-460d-9aa0-9caf3999773b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import s3fs\n",
+    "\n",
+    "s3 = s3fs.S3FileSystem(anon=True)\n",
```

Comment:
Why are we using s3fs here? I think we should be doing it the same way we do it in the dask dataframe example right? Where we just directly use read_csv into the dask dataframe? If there is a reason we have to use s3fs we should say why. My hunch is this is just an older example though.